recipes-core/images/seapath-guest-common.inc

```diff
@@ -14,6 +14,9 @@ IMAGE_INSTALL_append = " \
     net-snmp-server \
     net-snmp-client \
     net-snmp-dev \
+    ethtool \
+    chrony \
```

Comment:
chrony and systemd-timesyncd are in conflict.
I suggest to disable systemd-timesyncd:
Create the file *recipes-core/systemd/systemd_%.bbappend* with the following content:
```
# Disable systemd-timesyncd in guest images (chrony with KVM PTP is used instead)
do_install_append_votp-guest() {
    rm -rf ${D}${sysconfdir}/systemd/system/sysinit.target.wants/systemd-timesyncd.service
}
```